config/_default/menus/menus.zh-cn.toml

```diff
@@ -44,7 +44,7 @@
 [[cta]]
     identifier = "cta"
     name = "下载"
-    url = "/docs/download/"
+    url = "/docs/install/"
```

Comment:
is there a way to have both? For SEO/search? Both are very common terms for the same thing.